acme/src/acme/client.py

```diff
@@ -503,17 +503,18 @@ class ClientNetwork:
 
     """Initialize.
 
-    :param josepy.JWK key: Account private key
+    :param josepy.JWK key: Account private key. Required to use .post().
     :param messages.RegistrationResource account: Account object. Required if you are
             planning to use .post() for anything other than creating a new account;
             may be set later after registering.
-    :param josepy.JWASignature alg: Algorithm to use in signing JWS.
+    :param josepy.JWASignature alg: Algorithm to use in signing JWS. Required to use .post().
```

Comment:
Why this change, and not keep the default? I think since we used to set a default, if we want to change this it'll have to wait for a major release.

Reply:
Good point, thanks. I was chasing mypy errors and got it in my head that if `key` was None, so should `alg` be. But of course it's fine for `alg` to have its default value when `key` is None, because it's not used. 👍🏻